spec/system/task_spec.rb

```diff
@@ -58,23 +50,20 @@
   end
 
   describe 'Task編集' do
+    let(:done_task) { FactoryBot.create(:task, project_id: project.id,
+                                        status: :done,
+                                        completion_date: Time.current.yesterday) }
```

Comment:
traitを定義して使うようにしましょう。同じ内容のテストデータの定義を使いまわすことができます。
```suggestion
    let(:done_task) { create(:task, :done) }
```

Reply:
36cd46a15eb8a861f4d656ebf0bfc23014563323 でtraitを定義しました。